### ISSUE TITLE
Bug: (fill in)

### ISSUE THREAD

**Issue by cvez** [eslint/eslint#16949]
### Environment

Node version: v16.19.0
npm version: 9.4.2
Local ESLint version: 3.32.0
Global ESLint version: 2020
Operating System: windows 11

### What parser are you using?

vue-eslint-parser

### What did you do?

<details>
<summary>Configuration</summary>

```
 Failed to load config "@vue/prettier/@typescript-eslint" to extend from.
Referenced from: C:\Dev\Casino-Titan-Web\.eslintrc.js
```
</details>

```js
      files: ['**/*.vue'],
      extends: [
        'plugin:vue/vue3-essential',
        '@vue/typescript/recommended',
        '@vue/prettier',
        '@vue/prettier/@typescript-eslint'
      ],
      parser: 'vue-eslint-parser',
      parserOptions: {
        sourceType: 'module',
        ecmaVersion: 2020,
        parser: '@typescript-eslint/parser',
      },
      rules: {
        'import/no-default-export': 'off',
        'import/no-internal-modules': 'off',
      },
    },
```

### What did you expect to happen?

Expected when running yarn lint to get no errors when upgrading plugins to newer versions

### What actually happened?

 Error: Failed to load config "@vue/prettier/@typescript-eslint" to extend from.
Referenced from: C:\Dev\Casino-Titan-Web\.eslintrc.js
    at configMissingError (C:\Dev\Casino-Titan-Web\node_modules\gulp-eslint\node_modules\eslint\lib\cli-engine\config-array-factory.js:265:9)
    at ConfigArrayFactory._loadExtendedShareableConfig (C:\Dev\Casino-Titan-Web\node_modules\gulp-eslint\node_modules\eslint\lib\cli-engine\config-array-factory.js:826:23)
    at ConfigArrayFactory._loadExtends (C:\Dev\Casino-Titan-Web\node_modules\gulp-eslint\node_modules\eslint\lib\cli-engine\config-array-factory.js:731:25)
    at ConfigArrayFactory._normalizeObjectConfigDataBody (C:\Dev\Casino-Titan-Web\node_modules\gulp-eslint\node_modules\eslint\lib\cli-engine\config-array-factory.js:660:25)
    at _normalizeObjectConfigDataBody.next (<anonymous>)
    at ConfigArrayFactory._normalizeObjectConfigData (C:\Dev\Casino-Titan-Web\node_modules\gulp-eslint\node_modules\eslint\lib\cli-engine\config-array-factory.js:596:20)
    at _normalizeObjectConfigData.next (<anonymous>)
    at ConfigArrayFactory._normalizeObjectConfigDataBody (C:\Dev\Casino-Titan-Web\node_modules\gulp-eslint\node_modules\eslint\lib\cli-engine\config-array-factory.js:699:25)
    at _normalizeObjectConfigDataBody.next (<anonymous>)
    at ConfigArrayFactory._normalizeObjectConfigData (C:\Dev\Casino-Titan-Web\node_modules\gulp-eslint\node_modules\eslint\lib\cli-engine\config-array-factory.js:596:20)

### Participation

- [ ] I am willing to submit a pull request for this issue.
